components/molecules/getAddress/index.js

```diff
@@ -3,136 +3,158 @@
  */
 
 // React
-import { forwardRef, useImperativeHandle, useState } from 'react'
-import { func, string } from 'prop-types'
+import { useRef, useState } from 'react'
+import { func, object, oneOf, number, string } from 'prop-types'
 
 // UI
-import { Button, FormField, FormLabel, SelectField, validatorPostCode } from '../../'
-
-export const GetAddress = forwardRef(
-  (
-    { apiKey, error, change, form, handleFindAddress, handlePopulateAddress, selectAddress },
-    ref
-  ) => {
-    console.log(form)
-    const [loading, setLoading] = useState(false)
-    const [addresses, setAddresses] = useState({ data: [] })
-
-    useImperativeHandle(ref, () => ({
-      removeWhitespace: postcode => {
-        return postcode.replace(/\s/g, '')
-      },
-
-      validatePostcode: postcode => {
-        validatorPostCode(postcode)
-      },
-
-      getAddress: postcode => {
-        setLoading(true)
-        // let data = addresses.data
-        // data = ADDRESS_DATA.addresses
-        // setAddresses({ ...addresses, data: [...data] })
-
-        // Fetch
-        window
-          .fetch(`https://api.getaddress.io/find//${postcode}?api-key=${apiKey}`)
-          .then(response => {
-            if (!response.ok) {
-              return false
-            }
-            return response.json()
-          })
-          .then(data => {
-            setAddresses({ ...addresses, data: data.addresses })
-          })
-          .catch(error => {
-            console.error('Error', error)
-          })
-      }
-    }))
-
-    const input = () => (
-      <FormLabel label='Postcode'>
-        <FormField onChange={change} name='postcode' value={form.postcode} />
-      </FormLabel>
-    )
-
-    const button = () => (
-      <Button content='Find your address' context='primary' onClick={handleFindAddress} size='lg' />
-    )
-
-    const postcodeAddresses = () => {
-      if (loading) {
-        if (addresses.data.length > 0) {
-          // Reduce the array to conform to the select
-          const reduced = addresses.data.reduce((acc, cur, i) => {
-            acc.push({ text: cur, value: i + 1 })
-            return acc
-          }, [])
-
-          reduced.unshift({ disabled: true, text: 'Select address', value: '' })
-
-          return (
-            <>
-              <p />
-              <FormLabel label='Select your address'>
-                <SelectField
-                  onChange={handlePopulateAddress}
-                  name='addresses'
-                  options={reduced}
-                  value={selectAddress}
-                />
-              </FormLabel>
-            </>
-          )
-        }
-      }
+import {
+  Button,
+  Dropdown,
+  FormError,
+  FormField,
+  FormLabel,
+  GetAddressService,
+  InputGroupAddon,
+  Shimmer,
+  SIZE,
+  validatorPostCode
+} from 'components'
+import styled, { css } from 'styled-components'
+// test postCode: SW19 2EZ
```

Comment:
I think you forgot to remove comments. If it's for testing, then you can put it in test or story file.